### PR TITLE
Add swagger and API annotations

### PR DESCRIPTION
**Pull request by Moonchopper**

### CODE REVIEW

**Comment by carltonwhitehead** [comment]
From the Travis CI build:
```
[INFO] There are 11 checkstyle errors.
[ERROR] ConerDropwizardApplication.java[35:8] (imports) UnusedImports: Unused import - java.io.IOException.
[ERROR] EventRegistrationResource.java[9] (imports) AvoidStarImport: Using the '.*' form of import should be avoided - com.wordnik.swagger.annotations.*.
[ERROR] EventRegistrationResource.java[58] (sizes) LineLength: Line is longer than 120 characters (found 134).
[ERROR] EventRegistrationResource.java[60] (sizes) LineLength: Line is longer than 120 characters (found 124).
[ERROR] EventRegistrationResource.java[61] (sizes) LineLength: Line is longer than 120 characters (found 147).
[ERROR] EventRegistrationsResource.java[10] (imports) AvoidStarImport: Using the '.*' form of import should be avoided - com.wordnik.swagger.annotations.*.
[ERROR] EventRegistrationsResource.java[63] (sizes) LineLength: Line is longer than 120 characters (found 166).
[ERROR] EventRegistrationsResource.java[65] (sizes) LineLength: Line is longer than 120 characters (found 149).
[ERROR] EventResource.java[8] (imports) AvoidStarImport: Using the '.*' form of import should be avoided - com.wordnik.swagger.annotations.*.
[ERROR] EventsResource.java[9] (imports) AvoidStarImport: Using the '.*' form of import should be avoided - com.wordnik.swagger.annotations.*.
[ERROR] ConerDropwizardApplicationTest.java[10] (imports) AvoidStarImport: Using the '.*' form of import should be avoided - io.federecio.dropwizard.swagger.*.
```
Please take care of these style errors. You can run `mvn checkstyle:check` to check locally

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "dropwizard-swagger"]
```
Comment (carltonwhitehead) [inline .gitmodules]: Why did you add this as a submodule? The dependency declaration in the pom.xml should suffice

Comment (Moonchopper) [inline .gitmodules]: Shouldn't be there - thought I had taken it out. I'll need to check it when I get home.

Comment (Moonchopper) [inline .gitmodules]: Oh! Never mind, forgot that I didn't update .gitmodules - had to do something funky with it prior to, forgot it afterwards.

Comment (Moonchopper) [inline .gitmodules]: I cannot remember the method by which I added this, but I have removed the submodule from .gitmodules - I believe this should suffice.
